### PR TITLE
feat: Apply prettier to generated markdown files for improved formatting

### PR DESCRIPTION
**Pull request by AlanFoster**
Applies prettier to generated markdown files to enforce formatting consistency, and improve the readability of generated tables within a text editor.

### CODE REVIEW

File: examples/docs/complex.schema.md [hunk 1]
```diff
@@ -3,297 +3,211 @@ template: reference
...
+| --------------------------- | --------- | ------------ | -------- | ------------------------------------------ |
+| [and](#and)                 | complex   | Optional     | No       | Complex References (this schema)           |
+| [or](#or)                   | complex   | Optional     | No       | Complex References (this schema)           |
+| [refabstract](#refabstract) | `object`  | **Required** | No       | Complex References (this schema)           |
+| [reflist](#reflist)         | Simple    | Optional     | No       | Complex References (this schema)           |
+| [refnamed](#refnamed)       | Simple    | Optional     | No       | Complex References (this schema)           |
+| [xor](#xor)                 | complex   | Optional     | No       | Complex References (this schema)           |
+| `int.*`                     | `integer` | Pattern      | No       | Complex References (this schema)           |
+| `str.*`                     | `string`  | Pattern      | No       | Complex References (this schema)           |
+| `*`                         | any       | Additional   | Yes      | this schema _allows_ additional properties |
```
Comment (AlanFoster) [inline examples/docs/complex.schema.md]: 👀

**Comment by trieloff** [comment]
Hi @AlanFoster – thanks for the PR, this is a great idea, I wish I had it myself. Could you sign the Adobe CLA here http://opensource.adobe.com/cla.html – so that we get the last green checkmark and can merge the PR?

**Comment by AlanFoster** [comment]
@trieloff I've signed the CLA, but the bot still doesn't seem happy

**Comment by trieloff** [comment]
Ok, I'll merge anyway.

**Comment by trieloff** [comment]
Thank you, @AlanFoster!

**Pull request merged**